esmvalcore/cmor/_fixes/fix.py

```diff
@@ -7,23 +7,23 @@
 
 
 class Fix:
-    """
-    Base class for dataset fixes.
-    """
-    def __init__(self, vardef):
+    """Base class for dataset fixes."""
+    def __init__(self, vardef, extra_facets={}):
```

Comment:
This will give you a Codacy error that a `dict` is an unsafe default, because the `dict` that is used as the default value here, is created when the function is loaded from the module, not when the function is called. That means that if you would modify `extra_facets` inside the function, any subsequent calls of the function will use the modified `extra_facets`, potentially leading to weird bugs.